<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Anonymizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="xmlDataButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>XML File + Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dicomDirButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DICOMDIR File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="anonymizedFilesPathButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>29</y>
      <width>40</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dicomFilesButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DICOM Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Videos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Endoscopy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MRI / US</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="resetDefaultPathButton">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>29</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>toto</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="destinationNoMatch">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>11</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actual destination â‰  Default database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="useViewerDBButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>9</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use default DB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="anonymizedFilesPathText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>406</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination folder for anonymous files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>69</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidgetProgress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPlainTextEdit" name="progressTextDetailled">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>477</width>
        <height>267</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPlainTextEdit" name="progressTextSummary">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>477</width>
        <height>267</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="remainingAnonLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>111</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxBot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>402</y>
      <width>481</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: no;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="cancelCurrentAnonymisationButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>25</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="progressCurrentAnonLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current Anonymisation</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>481</width>
       <height>91</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QLineEdit" name="NameEditNtoID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Enter name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="FirstnEditNtoID">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Enter firstname</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>12</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>122</x>
         <y>12</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Firstname</string>
       </property>
      </widget>
      <widget class="QPushButton" name="NameToIDButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>28</y>
         <width>28</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="IdEditNtoID">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>262</x>
         <y>12</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="dlButtonNtoID">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Download </string>
       </property>
      </widget>
      <widget class="QLabel" name="nonAnonymousDocsLabelNtoID">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Non-anonymous additionnal documents</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ulButtonNtoID">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QLineEdit" name="NameEditIDtoN">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="FirstnEditIDtoN">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>12</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>12</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Firstname</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="IdEditIDtoN">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>12</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="IDToNameButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>28</y>
         <width>28</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="nonAnonymousDocsLabelIDtoN">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Non-anonymous additionnal documents</string>
       </property>
      </widget>
      <widget class="QPushButton" name="dlButtonIDtoN">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Download </string>
       </property>
      </widget>
      <widget class="QPushButton" name="ulButtonIDtoN">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QProgressBar" name="anonymisationProgressBar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressCurrentFileLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current File</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="fileProgressBar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
